### ISSUE TITLE
No crop?

### ISSUE THREAD

**Issue by nathanhornby** [ccorcos/meteor-clientside-image-manipulation#9]
Is there no flag to crop the image, does it only resize to one dimension? I'm assuming there must be a way otherwise it would be of quite limited use?

**Comment by ccorcos** [comment]
It resizes as a containing box. wouldn't be too hard to crop. Its just a little blind...

**Comment by nathanhornby** [comment]
Would be great addition - only thing I felt was missing from the package (for my needs anyway)!

**Comment by ccorcos** [comment]
yeah, i primarily use this for compression and fixing the orientation of the image. cropping would probably need an interface though.

**Comment by nathanhornby** [comment]
I mean cropping to the provided dimensions, not a user crop.

Ie if I say i want a 200x300 image it should be center-cropped to that size.

Does that make sense?

**Comment by ccorcos** [comment]
wouldn't be hard just to crop right here:

https://github.com/ccorcos/meteor-clientside-image-manipulation/blob/master/lib/processImage.coffee#L86
